install/kubernetes/helm/istio/values-istio-auth-certmanager.yaml

```diff
@@ -0,0 +1,20 @@
+# This is used to generate istio.yaml
```

Comment:
Not sure what this file is doing - we need to remove all the other similar files as well, please remove from the PR.

Reply:
done